src/main/java/org/codehaus/plexus/archiver/UnArchiver.java

```diff
@@ -72,6 +73,25 @@ void extract( String path, File outputDirectory )
      */
     void setOverwrite( boolean b );
 
+    /**
+     * Get component which rewrites the target path of each unpacked file.
+     *
+     * @return {@link FileMapper} to be used for rewriting each target path, or {@code null} if no rewriting shall happen.
+     *
+     * @since 3.7
```

Comment:
We use three digit versions now, so please use `3.7.0`

Reply:
ok I will fix that

Reply:
Fixed by https://github.com/codehaus-plexus/plexus-archiver/pull/100/commits/67568e1d512171aff8a3de7b6ae3ceeea7e2091a.